Add training signup

szczys opened the pull request:
- Add button to the landing page
- Add link in the footer

szczys committed:
add training sign-up button to landing page

Signed-off-by: Mike Szczys <mike@golioth.io>

szczys committed:
add training sign-up to the footer

Signed-off-by: Mike Szczys <mike@golioth.io>

cdwilson commented on website/docusaurus.config.js:
```diff
@@ -139,6 +139,10 @@ const config = {
                 href: 'https://codespaces.new/golioth/zephyr-training',
               },
+              {
+                label: 'Sign up for live Zephyr Training course',
```
Maybe "Sign up for [a|our] live Zephyr Training course" or "Sign up for live Zephyr Training"?

szczys replied:
Good catch! I went with: "Take our live Zephyr Training course"

cdwilson reviewed (approved):
Looks great!

I'm fine merging this as-is, but the only thing that may be worth mentioning for the future is that for smaller screens, this can be hidden unless you scroll down, which may cause people to miss it:

<img width="1280" alt="image" src="https://github.com/golioth/developer-training/assets/14631/18b66480-aa27-492a-b440-4d7e18819f40">

szczys committed:
change hero background color to grey

Change hero background color from coral to grey so you don't get a flash of
coral while the background image is loading.

Signed-off-by: Mike Szczys <mike@golioth.io>
